exercise_name3.rb

```diff
@@ -0,0 +1,5 @@
+puts "What is the quote?"
+quote = gets
+puts "Who said it?"
+name = gets.chomp
+puts "#{name} says,  #{quote}"
```

Comment:
I think that exercise requires the quote to be in quotation marks, how could you do this?